src/leo-server-app/src/models/log.ts

```diff
@@ -0,0 +1,19 @@
+import mongoose from "mongoose";
+
+const Schema = mongoose.Schema;
+
+const logSchema = new Schema({
```

Comment:
if we batch commands, maybe we should have a id for it as well?

i.e if we package like 10 commands together to be sent at a time, we should have some sort of way of querying those commands

Reply:
also might be good to know the target (neudose vs test sat)

Reply:
oh dang yeah that's true, can we have this as part of a future task/pr because i feel like it needs to be scoped out more and this pr mainly touches on the user management stuff

Reply:
sure, can you just make it a issue so its something we can look at later

Reply:
yeah